deploy/cluster-manager/config/rbac/cluster_role.yaml

```diff
@@ -9,7 +9,7 @@ rules:
   verbs: ["create", "get", "list", "update", "watch", "patch", "delete", "deletecollection"]
 - apiGroups: [""]
   resources: ["secrets"]
-  verbs: ["list", "watch", "update", "patch", "delete"]
+  verbs: ["get", "list", "watch", "update", "patch", "delete"]
```

Comment:
get is not needed, right?

Reply:
the cluster role here is configured for cluster-manager, I think get for the following secrets is still needed, I just changed it back to several days ago.